sdk/python/kfp/compiler/_op_to_template.py

```diff
@@ -250,8 +250,38 @@ def _op_to_template(op: BaseOp):
         if processed_op.pod_labels:
             template['metadata']['labels'] = processed_op.pod_labels
     # retries
-    if processed_op.num_retries:
-        template['retryStrategy'] = {'limit': processed_op.num_retries}
+    if processed_op._node_retry_strategy:
+        retry_strategy = op._node_retry_strategy
+        retry_strategy_dict = {}
+
+        if retry_strategy.num_retries:
+            retry_strategy_dict['limit'] = retry_strategy.num_retries
+
+        backoff_dict = {}
+        if retry_strategy.backoff_duration:
```

Comment:
Do you think it's better to use Argo terms here ('duration') so that it's easier to memorize?

Reply:
Correct me if I'm wrong, but Argo also has `backoff` here: `retry_strategy.backoff.duration`. I've just flattened/inlined the backoff sub-structure.